wb/nm_helper/virtual_devices/virtual_devices.py

```diff
@@ -141,16 +152,23 @@ def _common_connection_added_handler(self, *args, **kwargs):
         # Finally it receives normal messages after garbage
         connection_path = args[0]
         if kwargs["sender"] in self._bus.list_names():
+            self._mutex.acquire()
             self._common_connections[connection_path] = CommonConnection(
                 self, self._mqtt_client, self._bus, connection_path, self._logger
             )
+            self._mutex.release()
```

Comment:
```suggestion
            with self._mutex:
                self._common_connections[connection_path] = CommonConnection(
                    self, self._mqtt_client, self._bus, connection_path, self._logger
                )
```

Здесь и во всех остальных местах. Это важно, потому что иначе мьютекс не освободится, если в середине выпадет исключение